doc/source/reference/offset_frequency.rst

```diff
@@ -33,8 +33,8 @@ Methods
 
     DateOffset.apply
     DateOffset.copy
-    DateOffset.isAnchored
-    DateOffset.onOffset
+    DateOffset.is_anchored
+    DateOffset.is_on_offset
```

Comment:
If you want a "quick" (ahum) solution without needing to dive deep into sphinx, I would just leave them both in the above autosummary. I *think* that should also solve it

Reply:
> If you want a "quick" (ahum) solution without needing to dive deep into sphinx, I would just leave them both in the above autosummary

Sounds good to me, thanks